{"uid":"7d2ab8b9e060e9ae","name":"正向用例：登录后成功创建订单","fullName":"test_cases.test_order_flow.TestOrderFlow#test_login_and_create_order_success","historyId":"e2d0c09de0394f8b9be8a4057515c17b","time":{"start":1758437187628,"stop":1758437187635,"duration":7},"description":"测试用户登录后能否成功下单","descriptionHtml":"<p>测试用户登录后能否成功下单</p>\n","status":"broken","statusMessage":"Exception: 登录失败: {\n  \"code\": 401,\n  \"msg\": \"用户名或密码错误\"\n}","statusTrace":"self = <test_order_flow.TestOrderFlow object at 0x7f9da6da39e8>\n\n    @allure.story(\"用户成功下单流程\")\n    @allure.title(\"正向用例：登录后成功创建订单\")\n    def test_login_and_create_order_success(self):\n        \"\"\"测试用户登录后能否成功下单\"\"\"\n        # 准备测试数据\n        username = \"test_user\"\n        password = \"test1234560\"\n        item_id = 1\n        quantity = 2\n    \n        # 执行业务流程\n>       order_id = self.order_business.create_order_flow(username, password, item_id, quantity)\n\ntest_cases/test_order_flow.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nbusiness/order_business.py:44: in create_order_flow\n    session = self.login_and_get_session(username, password)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <business.order_business.OrderBusiness object at 0x7f9da67b8da0>, username = 'test_user', password = 'test1234560'\n\n    def login_and_get_session(self, username, password):\n        \"\"\"\n        业务逻辑：登录并获取已认证的session\n        :return: 携带token的session对象\n        \"\"\"\n        # 调用API Object层\n        login_api = LoginApi()\n        response = login_api.login(username, password)\n    \n        if response.status_code != 200:\n>           raise Exception(f\"登录失败: {response.text}\")\nE           Exception: 登录失败: {\nE             \"code\": 401,\nE             \"msg\": \"用户名或密码错误\"\nE           }\n\nbusiness/order_business.py:22: Exception","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":7,"retriesStatusChange":true,"beforeStages":[{"name":"setup","time":{"start":1758437187627,"stop":1758437187627,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"setup::0","time":{"start":1758437187678,"stop":1758437187679,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"订单流程"},{"name":"epic","value":"电商平台"},{"name":"story","value":"用户成功下单流程"},{"name":"parentSuite","value":"test_cases"},{"name":"suite","value":"test_order_flow"},{"name":"subSuite","value":"TestOrderFlow"},{"name":"host","value":"iZ2ze32pafdbs32qn8pvbwZ"},{"name":"thread","value":"20847-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_cases.test_order_flow"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"94a1d315ea177673","status":"passed","time":{"start":1758344266027,"stop":1758344266038,"duration":11}},{"uid":"3351a4d01a8d0206","status":"passed","time":{"start":1758343722425,"stop":1758343722434,"duration":9}},{"uid":"631ac9fcc3c456fc","status":"passed","time":{"start":1758343336406,"stop":1758343336420,"duration":14}},{"uid":"4a2bbb6c7a3c9083","status":"passed","time":{"start":1758343272097,"stop":1758343272108,"duration":11}},{"uid":"c72932781a3874cf","status":"passed","time":{"start":1758343109643,"stop":1758343109656,"duration":13}},{"uid":"6ede740150d4901d","status":"passed","time":{"start":1758342979629,"stop":1758342979642,"duration":13}},{"uid":"56420bee4ba3312e","status":"broken","statusDetails":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8000): Max retries exceeded with url: /api/login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc0a5de5c18>: Failed to establish a new connection: [Errno 111] Connection refused',))","time":{"start":1758342912694,"stop":1758342912699,"duration":5}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"7d2ab8b9e060e9ae.json","parameterValues":[]}